firmware/user/modes/mode_roll.c

```diff
@@ -282,7 +282,7 @@ void ICACHE_FLASH_ATTR initializeConditionsForODE(uint8_t Method)
             roll.rhs_fun_ptr = &dnxdampedpendulum;
             roll.pendulumParameters = (pendParam)
             {
-                .damping = 0.9,
+                .damping = 0.01,
```

Comment:
The right value is somewhere between 0.01, where the ball can oscillate pretty wildly, and 0.9, where it feels calmer. I might go with 0.15 or 0.25, which are both used for three ball